<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifChambre</class>
 <widget class="QDialog" name="ModifChambre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit
{

    border: 1px solid gray;
    border-radius: 4px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDialog
{
			background-image: url(:/projetimages/projetimages/a.jpg);


}
*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }

#label{
color:white;
background:transparent;

}

#numchmc_txt
{
color:white;
background:transparent;

}


#statutchambre_txt{
color:white;
background:transparent;

}

#label_3{
color:white;
background:transparent;

}





QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover
{

	color:white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QPushButton" name="modfifierstatutchambre_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier statut</string>
    </property>
   </widget>
   <widget class="QLabel" name="statutchambre_txt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prixchambremc_txt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>218</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Num√©ro de chambre :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numchmc_txt">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="modifierprixchambre_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
